api/graphql/schema.graphql

```diff
@@ -49,3 +53,16 @@ type Query {
   person(id: ID): Person
   community(id: ID, slug: String): Community
 }
+
+input MeInput {
+  name: String
+  email: String
+  avatarUrl: String
+  bannerUrl: String
+  bio: String
+  url: String
```

Comment:
This was the set that I was pretty confident we would continue to directly update as simple strings. I think we might want to create separate mutations for tags, password, and social media links. And then there are a few other fields that are not used anymore.